ibm/service/power/resource_ibm_pi_network.go

```diff
@@ -438,3 +448,39 @@ func getIPAddressRanges(ipAddressRanges []interface{}) []*models.IPAddressRange
 	}
 	return ipRanges
 }
+func waitForPERWorkspaceActive(ctx context.Context, client *instance.IBMPIWorkspacesClient, id string, timeout time.Duration) (interface{}, error) {
+	stateConf := &retry.StateChangeConf{
+		Pending:    []string{State_Inactive, State_Configuring},
+		Target:     []string{State_Active},
+		Refresh:    isPERWorkspaceRefreshFunc(client, id),
+		Timeout:    timeout,
+		Delay:      10 * time.Second,
```

Comment:
Is the PER workspace becoming active a fast operation? Checking every 10 seconds seems like too often. Timeout seems too short also.

Reply:
@ismirlia Yes, it takes less then 30 seconds for PER to be configure in a WS.